libs/SalesforceNetwork/SalesforceNetwork.xcodeproj/project.pbxproj

```diff
@@ -74,6 +74,10 @@
 		6FD16C7D1AA4EF4E001628F3 /* CSFAvailability.h in Headers */ = {isa = PBXBuildFile; fileRef = 827DD4751A7C006F00519F03 /* CSFAvailability.h */; settings = {ATTRIBUTES = (Public, ); }; };
 		6FD16C951AA4EF88001628F3 /* SalesforceSDKCore.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 6FD16C901AA4EF73001628F3 /* SalesforceSDKCore.framework */; };
 		6FD16CAB1AA4EFB5001628F3 /* SalesforceOAuth.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 6FD16CA61AA4EFA8001628F3 /* SalesforceOAuth.framework */; };
+		820053F81ACDBE7200C96578 /* NSDate+ActionValue.h in Headers */ = {isa = PBXBuildFile; fileRef = 820053F61ACDBE7200C96578 /* NSDate+ActionValue.h */; };
```

Comment:
This guy somehow wasn't in the project, but CocoaPods was grabbing it, then complaining about a bug (which you'll see the fix in the file below).